### PR TITLE
La til tester for Tilgangspakker og visning gitt rolle

### PR DESCRIPTION
**Pull request by ivartryti**
## Related Issue(s)
- #1865 

## Verification
- [ ] **Your** code builds clean without any errors or warnings
- [ ] Manual testing done (required)
- [ ] Relevant automated test added (if you find this hard, leave it and we'll help out)
- [ ] All tests run green

## Documentation
- [ ] User documentation is updated with a separate linked PR in [altinn-studio-docs.](https://github.com/Altinn/altinn-studio-docs) (if applicable)

## Summary by CodeRabbit

* **Tests**
  * Added comprehensive end-to-end tests for access management and delegation flows across multiple roles and service areas, covering delegation, visibility, permission checks, self-delegation edge cases, and a Hovedadministrator flow.
  * Added scenarios verifying regular users' limited visibility and inability to delegate packages they don't have.
  * Added a testdata suite to seed and manage data for these tests.

### CODE REVIEW

**Review by Nyeng — APPROVED** [review]

File: playwright/e2eTests/AccessManagement.spec.ts [hunk 1]
```diff
@@ -0,0 +1,252 @@
...
+      await accessManagementFrontPage.expectOthersWithRightsListToBeVisible();
+      await accessManagementFrontPage.clickUser('Iherdig Litteratur');
+    });
+
+    await test.step('Klikk "Gi fullmakt"', async () => {
+      await accessManagementFrontPage.clickGiFullmakt();
+    });
+
+    await test.step('Brukeren skal kunne til å gi fullmakt til "Tilgangsstyrer"', async () => {
+      await accessManagementFrontPage.GoToArea('Administrere tilganger');
```
[the remark this answers is not in the document]
Comment (ivartryti) [reply playwright/e2eTests/AccessManagement.spec.ts]: Fikset camelCase
